Tests/Invoke-DockerTag.Tests.ps1

```diff
@@ -8,24 +8,22 @@ Describe 'Tag docker images' {
         $script:moduleName = (Get-Item $PSScriptRoot\..\*.psd1)[0].BaseName
     }
 
-    Context 'Docker tag image with pester tag' {
+    $code = {
+        Write-Debug $Command
+        StoreMockValue -Key "mock" -Value $Command
+    }
 
-        BeforeEach {
-            Initialize-MockReg
-        }
+    BeforeEach {
+        Initialize-MockReg
+        Mock -CommandName "Invoke-Command" $code -Verifiable -ModuleName $script:moduleName
+    }
 
-        $code = {
-            Write-Debug $Command
-            StoreMockValue -Key "mock" -Value $Command
-        }
+    Context 'Docker tags public registry images' {
 
         It 'tags docker image with image name only' {
 
-            Mock -CommandName "Invoke-Command" $code -Verifiable -ModuleName $script:moduleName
-
             Invoke-DockerTag -SourceImage 'lalaland' -TargetImage 'lololand'
             Assert-MockCalled -CommandName "Invoke-Command" -ModuleName $script:moduleName
```

Comment:
If you want this exact line of code executed after each test case, consider moving it to an AfterEach block.

Reply:
Roger that.